lib/usage_data.js

```diff
@@ -37,34 +37,15 @@ exports.setUsageDataConfig = function (usageDataConfig) {
 // returns a Promise with the user's answer
 exports.askForUsageDataPermission = function () {
   return new Promise(function (resolve, reject) {
-    // Set up prompt settings
-    prompt.colors = false
-    prompt.start()
-    prompt.message = ''
-    prompt.delimiter = ''
+    const description = fs.readFileSync(path.join(__dirname, 'usage-data-prompt.txt'), 'utf8').trim()
 
-    const description = fs.readFileSync(path.join(__dirname, 'usage-data-prompt.txt'), 'utf8')
-
-    prompt.get([{
-      name: 'answer',
-      description: description,
-      required: true,
-      type: 'string',
-      pattern: /y(es)?|no?/i,
-      message: 'Please enter y or n',
-      ask: function () {
-        return process.stdout.isTTY
-      }
-    }], function (err, result) {
-      if (err) {
-        reject(err)
```

Comment:
I think we could add back this error catching and logging back with something like
```
    }]).then(
      answers => resolve(answers.usageData)
    )
  }).catch(
    error => console.log(error)
  )
```

Reply:
I think if there's an error then the promise should automatically be rejected, and the error should bubble up to the console anyway?

Reply:
Going to merge this for now and we can revisit this later if we need to.

Reply:
It outputs this if the error is not caught (if text file is missing): 
```
UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/Users/hannalaakso/www/govuk_prototype_kit/lib/usage-data-prompt.txt'
...
UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch()
```

If it's caught:
```
Error: ENOENT: no such file or directory, open '/Users/hannalaakso/www/govuk_prototype_kit/lib/usage-data-prompt.txt'
```